modules/admin-top-bar/module.php

```diff
@@ -88,22 +88,43 @@ private function add_frontend_settings() {
 		do_action( 'elementor/admin-top-bar/init', $this );
 	}
 
+	private function should_be_shown() {
+		$current_screen = get_current_screen();
+
+		if ( ! $current_screen ) {
+			return false;
+		}
+
+		$is_elementor_page = strpos( $current_screen->id ?? '', 'elementor' ) !== false;
+		$is_elementor_post_type_page = strpos( $current_screen->post_type ?? '', 'elementor' ) !== false;
+
+		return apply_filters(
+			'elementor/admin-top-bar/should-be-shown',
```

Comment:
IDK why, but this hook name sound weird ("should be shown"). maybe let's talk about it and try to find a different name?

Reply:
Done